skimage/measure/tests/test_fit.py

```diff
@@ -298,7 +298,8 @@ def test_ransac_geometric():
 
     # test whether estimated parameters equal original parameters
     assert_almost_equal(model0.params, model_est.params)
-    assert np.all(np.nonzero(inliers == False)[0] == outliers)
+    assert tuple(np.nonzero(np.atleast_1d(inliers == False))[0]) == outliers
```

Comment:
Why isn't the fix
```suggestion
    assert np.all(np.atleast_1d(inliers == False).nonzero()[0] == outliers)
```
? (based on NumPy's warning)